test/functional/apps/context/_date_nanos.js

```diff
@@ -75,17 +75,17 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.context.navigateTo(TEST_INDEX_PATTERN, 'AU_x3-TaGFA8no6Qjisd');
       await PageObjects.context.clickPredecessorLoadMoreButton();
       await PageObjects.context.clickSuccessorLoadMoreButton();
-      const actualRowsText = await docTable.getRowsText();
+      const actualRowsText = await dataGrid.getDataGridTableTimestamp();
       const expectedRowsText = [
-        'Sep 22, 2019 @ 23:50:13.2531233455',
-        'Sep 18, 2019 @ 06:50:13.0000001044',
-        'Sep 18, 2019 @ 06:50:13.0000001032',
-        'Sep 18, 2019 @ 06:50:13.0000001021',
-        'Sep 18, 2019 @ 06:50:13.0000001010',
-        'Sep 18, 2019 @ 06:50:13.000000001-1',
-        'Sep 18, 2019 @ 06:50:13.000000000-2',
-        'Sep 18, 2019 @ 06:50:12.999999999-3',
-        'Sep 19, 2015 @ 06:50:13.0001000011',
+        'Sep 22, 2019 @ 23:50:13.253123345',
+        'Sep 18, 2019 @ 06:50:13.000000104',
+        'Sep 18, 2019 @ 06:50:13.000000103',
+        'Sep 18, 2019 @ 06:50:13.000000102',
+        'Sep 18, 2019 @ 06:50:13.000000101',
+        'Sep 18, 2019 @ 06:50:13.000000001',
+        'Sep 18, 2019 @ 06:50:13.000000000',
+        'Sep 18, 2019 @ 06:50:12.999999999',
+        'Sep 19, 2015 @ 06:50:13.000100001',
```

Comment:
Do we know why these values changed?

Reply:
i had to delete the additional number at the end to make the values matched, i am not sure the cause, but it seems like new data grid table only render 9 digits now? @ananzh